src/test/java/org/ibp/api/java/impl/middleware/inventory/LocationValidatorTest.java

```diff
@@ -210,11 +237,236 @@ public void testvalidateInvalidLocation() {
 		this.errors = new MapBindingResult(new HashMap<String, String>(), LotUpdateRequestDto.class.getName());
 
 		try {
-			this.locationValidator.validateLocation(errors, LOCATION_ID);
+			this.locationValidator.validateLocation(errors, LocationValidatorTest.LOCATION_ID);
+		} catch (final ApiRequestValidationException e) {
+			assertThat(Arrays.asList(e.getErrors().get(0).getCodes()), hasItem("location.invalid"));
+			throw e;
+		}
+	}
+
+	@Test(expected = ApiRequestValidationException.class)
```

Comment:
It makes no sense to add the expected when the assertion is made when catching the exception. Applies to all tests

Reply:
I do it for the case when the validation does not fail. The Test will be ending successfully in that case.